api/app/Modules/Library/Http/Requests/Drafts/Lyrics/GetDraftLyricsRequest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Modules\Library\Http\Requests\Drafts\Lyrics;
+
+use App\Http\Requests\Request;
+use App\Modules\Library\Models\Track;
+use Illuminate\Validation\Rule;
+
+class GetDraftLyricsRequest extends Request
```

Comment:
Can you rename this to `ListDraftLyricsRequest` to make it clear that this is for the index route?